app/views/layout.mustache

```diff
@@ -45,6 +45,9 @@
     </PageMap>
     -->
     {{/has_pagemap}}
+    {{#include_survey}}
+    <script async="" defer="" src="//www.google.com/insights/consumersurveys/async_survey?site=5wdvu4vc5lhop3xbatxof6lzcm"></script>
```

Comment:
This is copy-pasted the recommended usage. Given that we host on https, I'm going to leave as-is.